src/app/qgisapp.cpp

```diff
@@ -15104,8 +15106,8 @@ void QgisApp::activateDeactivateLayerRelatedActions( QgsMapLayer *layer )
       mActionIdentify->setEnabled( true );
 
       QgsSettings settings;
```

Comment:
Just checking: is this still needed somewhere else in the function?

Reply:
good catch